packages/base/any/kernels/5.10-lts/patches/0042-net-phylink-sfp-Add-quirk-for-FINISAR-10G-modules.patch

```diff
@@ -0,0 +1,51 @@
+From 5c5d857711d80c3750c61103bfbaa547fa0e3f56 Mon Sep 17 00:00:00 2001
+From: Taras Chornyi <taras.chornyi@plvision.eu>
+Date: Mon, 1 Nov 2021 12:20:10 +0200
+Subject: [PATCH] net: phylink: sfp: Add quirk for FINISAR 10G modules
+
+Finisar FTLF8536P4BCL and FTLX1471D3BCL can operate
+in 1000Base-x mode as well
```

Comment:
Please reflow for 75 characters per line.

Reply:
will be fixed.
We will also up-streamed it soon.